dracut/parse-kickstart

```diff
@@ -43,7 +43,7 @@ from pykickstart.constants import *
 from collections import OrderedDict
 
 # Import the kickstart version.
-from pykickstart.version import DEVEL as VERSION
+from pykickstart.version import RHEL8 as VERSION
 
 # Import all kickstart commands as version-less.
```

Comment:
We should also replace the bootloader command below.

Reply:
Fixed, thank you!